resources/sqs-queues.go

```diff
@@ -0,0 +1,48 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+)
+
+type SQSQueue struct {
+	svc      *sqs.SQS
+	queueurl *string
```

Comment:
Sorry for being the pedantic one, but would you use `queueURL` to [follow golang style conventions][1]? The AWS SDK does not follow it, but we decided to follow it since it is the de-facto standard. 

[1]: https://github.com/golang/go/wiki/CodeReviewComments#initialisms